examples/webgpu_postprocessing_ssr.html

```diff
@@ -140,15 +140,27 @@
 			const metalRoughTexture = scenePass.getTexture( 'metalrough' );
 			metalRoughTexture.type = THREE.UnsignedByteType;
 
-			const customNormal = sample( ( uv ) => {
+			const sceneNormal = sample( ( uv ) => {
 
 				return colorToDirection( scenePassNormal.sample( uv ) );
 
 			} );
 
+			const sceneMetalness = sample( () => {
+
+				return scenePassMetalRough.r;
+
+			} );
+
+			const sceneRoughness = sample( () => {
+
+				return scenePassMetalRough.g;
+
+			} );
+
 			//
 
-			ssrPass = ssr( scenePassColor, scenePassDepth, customNormal, scenePassMetalRough, camera, true );
+			ssrPass = ssr( scenePassColor, scenePassDepth, sceneNormal, sceneMetalness, camera, sceneRoughness );
```

Comment:
Should we change the order here?

```js
ssr( scenePassColor, scenePassDepth, sceneNormal, sceneMetalness, sceneRoughness, camera );
```

Reply:
I think so too, but the camera is not yet an optional parameter, so the ideal would be to remove it.

Reply:
If you're planning to remove the camera I think this is better:

```ja
ssr( scenePassColor, scenePassDepth, sceneNormal, sceneMetalness, sceneRoughness = null, camera );
```

Less breaking changes.